dev/src/reference.ts

```diff
@@ -1405,46 +1675,353 @@ export class QueryOptions<
   }
 }
 
-/**
- * A Query refers to a query which you can read or stream from. You can also
```

Comment:
Isn't removing the Query class and changing it for QueryUtil a breaking change as well?